mongo/upsert_users.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+# Update users if database has been already created.
+# Users creation on init is left to `init_02_create_user.sh`
+BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
```

Comment:
Creative :+1: 
What about this, though?
```
echo "$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
echo $(readlink -f $(dirname "$BASH_SOURCE"))
john@scrappy:~$ bash bashy
/home/john
/home/john
```

Reply:
Ah Ah. I search for a simple way to do it. Thank you